src/style.css

```diff
@@ -197,109 +197,122 @@ footer .f-bottom {
-  width: 90vw;
-  min-height: 565px;
-  border: 2px solid rgb(223, 17, 113);
-  animation: 1s drop;
+.pop-up {
   display: none;
-  padding: 0 8px;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+  background-color: rgba(0, 0, 0, 0.5);
+  padding: 3%;
+  color: rgb(71, 68, 68);
+  overflow: auto;
+  height: 100%;
 }
 
-.close-button {
+.popup-details {
+  border: 1px solid black;
   position: absolute;
-  cursor: pointer;
-  outline: none;
-  margin-right: 4px;
+  display: flex;
+  justify-content: center;
+  border-radius: 10px;
+  margin: 3%;
+  flex-direction: column;
+  align-items: center;
   background-color: #fff;
-  color: #67798e;
-  border: none;
-  font-size: 30px;
-  font-weight: 600;
-  top: 0;
-  right: 0;
+  width: 86%;
+  padding-bottom: 5rem;
 }
 
-.add-comm {
+.flex {
   display: flex;
-  justify-content: center;
+  justify-content: space-between;
 }
 
-.inputDiv {
-  margin: 0 300px;
+h4 {
+  margin-right: 2rem;
+  align-items: center;
 }
 
-#name {
-  width: 200px;
-  color: #67798e;
-  margin-bottom: 16px;
-  padding: 5px;
+.flex h4 {
+  text-align: left;
 }
 
-#text {
-  width: 200px;
-  color: #67798e;
-  margin-bottom: 16px;
+.meal {
+  text-align: center;
 }
 
-.submit-btn {
-  padding: 5px 18px;
-  font-size: 15px;
-  margin-top: 10px;
-  color: red;
+.meal img {
+  width: 450px;
+  height: 300px;
 }
 
-.pop {
-  justify-content: center;
-  align-items: center;
-  background: #fff;
+.closebtn {
   position: relative;
-  top: 22px;
-  padding: 20px;
+  align-self: end;
+  text-decoration: none;
+  font-size: 2rem;
+  font-weight: bolder;
+  color: rgb(71, 68, 68);
+  margin-right: 4rem;
+  margin-top: 2rem;
+  cursor: pointer;
 }
 
-.popIt {
-  position: fixed;
-  top: 10px;
-  max-height: 100vh;
-  overflow: auto;
-  width: 100%;
+.no-comment {
+  font-style: italic;
 }
 
-.popup_image {
-  width: 100%;
-  display: flex;
-  justify-content: center;
+h2,
+h3 {
+  text-align: center;
 }
 
-.pop_image {
-  width: 200px;
-  margin: auto;
+#form {
+  flex-direction: column;
+  align-items: flex-start;
 }
 
-.instructions {
-  height: 100px;
-  margin-top: 20px;
-  width: 700px;
-  overflow: auto;
+::placeholder {
+  font-size: 1rem;
+  padding: 5px;
+}
 
-  /* background: #eeb3b3; */
-  font-size: 14px;
+.name-field,
+textarea {
+  margin-bottom: 1rem;
+  box-shadow: 1px 1px;
+  padding: 5px;
 }
 
-.pop_meal {
-  font-size: 20px;
+.name-field,
+textarea,
+.comment-btn {
+  border: 1px solid rgb(71, 68, 68);
+  border-radius: 5px;
 }
 
-.flex {
-  display: flex;
+.comment-btn,
+.add-comment-btn {
+  cursor: pointer;
+  padding: 10px 20px;
+  border: 1px solid black;
+  box-shadow: 3px 3px;
+  font-weight: bold;
+  color: darkgrey;
+  background-color: #fff;
+  border-radius: 5px;
 }
 
-.column {
-  flex-direction: column;
+.comment-sp {
+  font-style: italic;
+}
+
+.add-comment-btn:hover,
+.comment-btn:hover {
+  outline: none;
+  background: rgb(205, 158, 41);
+  color: white;
```

Comment:
We shall need to rearrange the classes to merge, this repo, later on before the merge is done